src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import CartItem from "./CartItem/CartItem";
+import styles from "./Drawer.module.scss";
+
+type DrawerProps = {
+    onClickCart: () => void;
+}
+
+const Drawer = (props: DrawerProps) => {
+    return (
+        <div className={styles.overlay}>
+            <div className={styles.drawer}>
+                <h2 className="d-flex justify-between mb-25">Корзина <button className={styles.closeBtn}><img onClick={props.onClickCart}
+                                                                                  src="/img/btn-close.svg" alt="Close"/></button></h2>
+                <div className={styles.items}>
+                    <CartItem />
```

Comment:
```suggestion
                    {props.map((item) => <CartItem  />)}
```
you should also add props for the CartItem component to be able to create a CartItem with the required data

Reply:
fixed